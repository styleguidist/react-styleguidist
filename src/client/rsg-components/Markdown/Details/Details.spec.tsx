import React from 'react';
import renderer from 'react-test-renderer';

import { Details, DetailsSummary } from './index';

describe('Markdown Details', () => {
	it('should render a Details', () => {
		const actual = renderer.create(
			<Details>
				<DetailsSummary>Solution</DetailsSummary>
				This is a hidden text.
			</Details>
		);

		expect(actual.toJSON()).toMatchSnapshot();
	});
});
