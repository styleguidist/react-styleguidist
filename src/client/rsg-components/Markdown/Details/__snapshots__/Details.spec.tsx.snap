// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown Details should render a Details 1`] = `
<details
  className="rsg--details-0"
>
  <summary
    className="rsg--summary-2"
  >
    Solution
  </summary>
  This is a hidden text.
</details>
`;
